src/component/reusable/Select.ts

```diff
@@ -0,0 +1,39 @@
+import { Attribute, RerenderListType } from "../../type/type";
+import { $ } from "../../utils/Dom";
+
+class Select {
+  attribute: Attribute;
+  options: string[];
+
+  constructor(attribute: Attribute, options: string[]) {
+    this.options = options;
+    this.attribute = attribute;
```

Comment:
```suggestion
    this.attribute = attribute;
    this.options = options;
```
매개변수를 받는 것과 순서를 일치하게 두는게 어떨까요?

Reply:
넵 순서 일치하게 수정하였습니다!